### ISSUE TITLE
Cross-language (R/python/H2O) contrastive explainers

### ISSUE THREAD

**Issue by pbiecek** [pbiecek/DALEX#62]
Prepare a use case (e.g. vigniette/blog/presentation) that compares models created in different languages,

for example native R (xgboost), native Python model (lightgbm) and some GBM from H2O on Titanic dataset.

The `reticulate` package might be used https://www.r-bloggers.com/the-reticulate-package-solves-the-hardest-problem-in-data-science-people/

**Comment by pbiecek** [comment]
https://medium.com/kaggle-nyc/gradient-boosting-decision-trees-xgboost-vs-lightgbm-and-catboost-72df6979e0bb
xgboost / gbm / catboost / random forest

**Comment by pbiecek** [comment]
https://github.com/pbiecek/DALEX_docs/blob/master/vignettes/Multilanguages_comparision.html
